TodoList/src/app/components/Navigator/index.js

```diff
@@ -0,0 +1,32 @@
+import { createStackNavigator, createBottomTabNavigator } from 'react-navigation';
+import Books from '../../screens/Books';
+import TodoList from '../../screens/ToDoList';
+
+const TabApp = createBottomTabNavigator({
+  Home: {
+    screen: Books,
+    navigationOptions: {
+      title: 'Books'
+    }
+  },
+  ToDoList: TodoList
+},
+{
+  initialRouteName: 'Home'
+},
+{
+  tabBarOptions: {
+    activeTintColor: 'red',
+    inactiveTintColor: 'gray'
+  }
+}
+);
+
+export default StackNavigator = createStackNavigator({
+  Home: {
```

Comment:
Move all routes names to an object in src/constants/routes.
The routes names are the keys of the first object you pass to each `createXNavigator`.
For example, the name of this route is `'Home'`